indico/modules/events/timetable/templates/_timetable.html

```diff
@@ -43,9 +43,13 @@
                 new BrowserHistoryBroker(),
             ].concat(extraArgs);
 
-            var timetable = new (Function.prototype.bind.apply(timetableClass, timetableArgs));
-            $('#timetable').html(timetable.draw());
-            timetable.postDraw();
+            if (Object.getOwnPropertyNames(timetableArgs[1]).length > 0) {
+                var timetable = new (Function.prototype.bind.apply(timetableClass, timetableArgs));
+                $('#timetable').html(timetable.draw());
+                timetable.postDraw();
+            } else {
+                $('#timetable').html($T("The timetable has not been filled yet."));
```

Comment:
is this only used in display mode? if not, what happens if you use one of the buttons to add e.g. a new contribution?